azure/storage/blob_test.go

```diff
@@ -50,7 +50,7 @@ func Test_blobSASStringToSign(t *testing.T) {
 }
 
 func TestGetBlobSASURI(t *testing.T) {
-	api, err := NewClient("foo", "YmFy", DefaultBaseUrl, "2013-08-15", true)
+	api, err := NewStorageClient("foo", "YmFy", DefaultBaseUrl, "2013-08-15", true)
```

Comment:
:+1: good catch. I left the storage clients alone (haven't really used them and didn't need to modify them) but this should be consistent.